.jscsrc

```diff
@@ -17,7 +17,7 @@
   "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
   "maximumLineLength": {
     "value": 80,
-    "allowUrlComments": true
+    "allowComments": true
```

Comment:
Which part did we need this for? It's pretty helpful to enforce 80 char limit in comment blocks.